### PR TITLE
Math upgrades

### PR DESCRIPTION
**Pull request by cindywu**
- [x] for inline-math show katex-render in tooltip and katex-editor inline
- [x] for block-math show-katex render below katex-editor

### CODE REVIEW

**Comment by cindywu** [comment]
Kapture 2020-10-07 at 20 40 07

File: app/assets/stylesheets/editor/math.scss [hunk 1]
```diff
@@ -2,37 +2,92 @@
...
   display: inline-block;
   width: auto;
   white-space: normal;
   position: relative;
-  font-family: monospace;
-}
 
-.math-inline > * {
-  display: inline;
+  .ProseMirror {
```
Comment (dennyluan) [inline app/assets/stylesheets/editor/math.scss:52]: smell: can fix this to just be explicit like the other lines

Comment (dennyluan) [reply app/assets/stylesheets/editor/math.scss:52]: e.g. `math.math-inline.ProseMirror-selectednode {`

File: app/javascript/components/PostEditor.js [hunk 2]
```diff
@@ -206,11 +204,11 @@ class PostEditor extends React.Component {
     const { isEditable } = this.state
     var menubar = isEditable ? menu : annotationMenu
     return (
-      <div>
+      <div className="post-editor-container">
```
Comment (dennyluan) [inline app/javascript/components/PostEditor.js:207]: why is this being added?

**Review by dennyluan — CHANGES_REQUESTED** [review]

**Comment by cindywu** [comment]
Closing this PR. We are migrating to Typescript https://github.com/jellypbc/poster/pull/404, so I will restart this PR to avoid the mess.

**Pull request closed without merging**